src/artikler/sok-sosialhjelp/komponenter/kommunesok/service/useTilgjengeligeKommunerService.ts

```diff
@@ -31,10 +41,12 @@ const useTilgjengeligeKommunerService = () => {
             url = "https://digisos.labs.nais.io" + url;
         } else if (window.location.origin.indexOf(".labs.nais.io") >= 0) {
             url = "https://sosialhjelp-soknad-api.labs.nais.io" + url;
+        } else if (window.location.origin.indexOf("localhost") >= 0) {
+            url = "http://localhost:8181" + url;
         } else {
```

Comment:
Jeg antar at både kode for heroku og codesandbox kan fjernes / skrives om, men tar dette i så fall i en egen PR. :)